libmscore/layout.cpp

```diff
@@ -792,12 +794,22 @@ static QPair<qreal, qreal> layoutAccidental(AcEl* me, AcEl* above, AcEl* below,
       for (int i = 0; i < lns; ++i) {
             Note* ln = leftNotes[i];
             int lnLine = ln->line();
-            qreal lnLedgerAdjust = 0.0;
-            if (lnLine <= -2 || lnLine >= staff->lines(tick) * 2)
-                  lnLedgerAdjust = ledgerAdjust;
```

Comment:
This is where I was adding space to avoid the ledger lines on "left notes", but it did this for all left notes on ledger lines.  In this PR, I refine this by avoiding setting lnLegderAdjust in cases where it isn't needed.